src/main/java/utility/Methods.java

```diff
@@ -0,0 +1,49 @@
+package utility;
+
+import tasks.Task;
+
+public class Methods {
```

Comment:
The name of this class does not describe what it is used for. Maybe consider naming it`Ui` instead since it seems to handle the functions for console output operations. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions)